components/alert/alert-toast.js

```diff
@@ -118,6 +137,19 @@ class AlertToast extends LitElement {
 		}
 	}
 
+	_closeTimerStart() {
+		if (!this.noAutoClose) {
+			const duration = this.buttonText ? 10000 : 4000;
```

Comment:
I think there are or will be scenarios where multiple things can cause `_closeTimerStart()` to run. For example, you could mouse over, then focus, then mouse out, then blur, causing 2 calls.

The best approach here might be to clear any existing timeouts before creating a new one.